.github/workflows/ebtree.yml

```
@@ -0,0 +1,51 @@
+# Build and store ebtree in cache
+
+name: EBTree
+on:
+  workflow_call:
+    inputs:
+      os:
+        type: string
+        required: true
+      # More a constant than a real input
+      ebtree_version:
+        type: string
+        default: 'v6.0.socio10'
+
+jobs:
+  ebtree:
+    name: Build EBTree
+    runs-on: ${{ github.event.inputs.os }}
```

```suggestion
    runs-on: ${{ github.inputs.os }}
```
No idea, just guessing based on the errors.